plugins/auto-sizes/tests/improve-sizes-test.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Tests for the improve sizes for Images.
+ *
+ * @package auto-sizes
+ * @group   improve-sizes
+ */
+
+class Tests_Improve_Sizes extends WP_UnitTestCase {
+
+	/**
+	 * Set up the environment for the tests.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+
+		switch_theme( 'twentytwentyfour' );
+	}
```

Comment:
We can set up the theme once before all tests rather than before each method.
```suggestion
	public static function set_up_before_class(): void {
		parent::set_up_before_class();

		switch_theme( 'twentytwentyfour' );
	}
```